server/models/Request.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require("mongoose");
+
+const RequestSchema = new mongoose.Schema({
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
+  sitterId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
+  startDate: {
+    type: Date,
+    required: true,
+  },
+  endDate: {
+    type: Date,
+    required: true,
+  },
+  accepted: {
```

Comment:
We can have an enum type object to keep track of the request status

```   status: {
        type: String,
        enum : ['pending','accepted', 'declined'],
        default: 'pending'
        lowercase: true,
    },
```

Reply:
fix it on PR #59